### PR TITLE
Upgrade Probot

### PR DESCRIPTION
**Pull request by jameesjohn**
This PR upgrades oppiabot-stale to the latest on probot-stale and uses the latest version of probot to resolve compatibility issues.

### CODE REVIEW

**Comment by jameesjohn** [comment]
PTAL @ankita240796 @kevinlee12. Thanks!

File: .github/workflows/ci.yml
Comment (kevinlee12) [inline .github/workflows/ci.yml]: Why are there so many node versions?

**Review by kevinlee12 — COMMENTED** [review]
I have just a comment, I'll defer to @ankita240796 for the i ndepth review.

File: .github/workflows/ci.yml
Comment (jameesjohn) [reply .github/workflows/ci.yml]: I think this is just done to ensure that the build supports all the different node versions.
It is part of the update from https://github.com/probot/stale

**Comment by kevinlee12** [comment]
got it. we aren’t running it on different versions, so it’s not necessary. I want us to get into the habit of running CI only when it’s necessary.

File: .github/workflows/ci.yml
Comment (ankita240796) [reply .github/workflows/ci.yml]: Why not just the latest?

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Line too long, break.

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Ditto.

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Ditto.

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Ditto.

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Ditto.

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Ditto.

File: test/stale.test.js
Comment (ankita240796) [inline test/stale.test.js]: Ditto here and below.

File: lib/schema.js
Comment (ankita240796) [inline lib/schema.js]: Ditto. Please fix everywhere.

File: .github/workflows/ci.yml
Comment (ankita240796) [reply .github/workflows/ci.yml]: Actually, I am fine with not having ci tests on this for repo for now since we are anyways planning to deprecate it.

**Review by ankita240796 — CHANGES_REQUESTED** [review]
Hi @jameesjohn, looks good. Few minor nits, PTAL. Thanks!

File: .github/workflows/ci.yml
Comment (jameesjohn) [reply .github/workflows/ci.yml]: Okay, I'd be removing this

File: lib/schema.js
Comment (jameesjohn) [reply lib/schema.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

File: test/stale.test.js
Comment (jameesjohn) [reply test/stale.test.js]: Done.

**Comment by jameesjohn** [comment]
PTAL @ankita240796

**Review by ankita240796 — APPROVED** [review]
LGTM, Thanks @jameesjohn! Just to confirm, have you tested this?

**Comment by jameesjohn** [comment]
> Just to confirm, have you tested this?

I have not been able to test this because this is a dependency in oppiabot and can only be tested after this PR has been merged then npm update will be get the latest version of this.

**Comment by ankita240796** [comment]
> > Just to confirm, have you tested this?
> 
> I have not been able to test this because this is a dependency in oppiabot and can only be tested after this PR has been merged then npm update will be get the latest version of this.

Makes sense. Will merge and deploy. Thanks!

**Pull request merged**
